hack/cert-manager/override-values.yaml

```
@@ -0,0 +1,31 @@
+trento-web:
+  ingress:
+    hosts:
+        # Adjust this to your FQDN
+      - host: demo.trento-project.io
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+    tls:
+      - secretName: trento-tls
```

as suggested above, just use the `annotations` field here
```suggestion
      - secretName: trento-tls
    annotations:
      cert-manager.io/cluster-issuer: letsencrypt-production
```
```